components/member-card/card.module.scss

```diff
@@ -33,8 +34,10 @@
 }
 
 .imgContainer {
-  width: 180px;
-  height: 180px;
+  // width: 180px;   og
+  // height: 180px;
+  width: 150px;
+  height: 150px;
```

Comment:
can we please use `rem`? I know there are places where we are using `px`, but I am making sure any new changes we have uses `rem`. There will be a refactor coming soon where the `px` will be changed to `rem`